aspnetcore/tutorials/first-mongo-app.md

```diff
@@ -263,69 +263,262 @@ The preceding web API controller:
 * Contains action methods to support GET, POST, PUT, and DELETE HTTP requests.
 * Calls <xref:Microsoft.AspNetCore.Mvc.ControllerBase.CreatedAtAction%2A> in the `Create` action method to return an [HTTP 201](https://www.rfc-editor.org/rfc/rfc9110#status.201) response. Status code 201 is the standard response for an HTTP POST method that creates a new resource on the server. `CreatedAtAction` also adds a `Location` header to the response. The `Location` header specifies the URI of the newly created book.
 
+## Configure JSON serialization options
```

Comment:
I moved this section before the testing section, because the testing section uses the configured JSON serialization options, e.g. `bookName` is mapped to `Name`.